extensions/clojure/syntaxes/Clojure.tmLanguage

```diff
@@ -7,6 +7,9 @@
 		<string>clj</string>
 		<string>cljs</string>
 		<string>clojure</string>
+		<string>cljc</string>
+		<string>cljx</string>
+		<string>edn</string>
```

Comment:
If you want to add file extensions for clojure, you have to do that `/extensions/clojure/package.json`. From `/extensions/clojure/syntaxes/Clojure.tmLanguage` we only take the colorizer information.